core/controllers/suggestion_test.py

```diff
@@ -999,6 +1000,65 @@ def test_suggestion_creation_with_valid_images(self):
                 ('file.svg', 'file.svg', raw_image), ))
         self.logout()
 
+    def test_suggestion_creation_fails_if_copy_images_fails(self):
+        self.save_new_skill(
```

Comment:
Can you add a comment explaining the test setup, i.e. what is being set up and why? Make for easier scanning of the test.

Reply:
Removed this code.